specs/v1.yaml

```diff
@@ -364,6 +377,89 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /v1/grants/unintegrated:
+    get:
+      summary: List Unintegrated Grants
+      description: |-
+        List all unintegrated grants for the provided API Key. An unintegrated grant,
+        is a grant made via a DAF provider that Chariot does not have a direct integration
+        with. These grants require the donor to go to their DAF portal, login, and manually
+        complete the grant. This API allows for paginating over many results.
+      operationId: list-unintegrated-grants
+      security:
+        - auth0:
+          - "read:grants"
+          api_key: []
+      parameters:
+        - name: pageLimit
+          in: query
+          description: the number of results to return; defaults to 10, max is 100
+          schema:
+            type: integer
+            default: 10
+        - name: pageToken
+          in: query
+          description: |-
+            A token to use to retrieve the next page of results. This is useful for paginating over
+            many pages of results. If set, all other arguments are expected to be kept the same as
+            previous calls and the value of this field should be from the nextPageToken in the previous response.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: "#/components/responses/ListUnintegratedGrantsResponse"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+  /v1/grants/unintegrated/{id}:
```

Comment:
working off comment above i think this should be: `/v1/unintegrated-grants/{id}`